pano2fisheye

```diff
@@ -0,0 +1,451 @@
+#!/bin/bash
+# 
+# Developed by Fred Weinhaus 8/11/2017 .......... revised 10/15/2018
+#
+# ------------------------------------------------------------------------------
+# 
+# Licensing:
+# 
+# Copyright © Fred Weinhaus
+# 
+# My scripts are available free of charge for non-commercial use, ONLY.
+# 
+# For use of my scripts in commercial (for-profit) environments or 
+# non-free applications, please contact me (Fred Weinhaus) for 
+# licensing arrangements. My email address is fmw at alink dot net.
+# 
+# If you: 1) redistribute, 2) incorporate any of these scripts into other 
+# free applications or 3) reprogram them in another scripting language, 
+# then you must contact me for permission, especially if the result might 
+# be used in a commercial or for-profit environment.
```

Comment:
Need to do this before integrating this into the code in any way. In fact, the Fisheye functionality should probably not be included as part of InContext.